NCI-GDC/gdc-frontend-framework
# PEAR-1688: remove score when creating gene or ssm set

## Description
Fixes an issue where gene/ssm sets are size 0. Adds case_filters to the gene/ssm setCreation query. I removed the score from these queries and added `useCreateTopNGeneSetFromFiltersMutation` and `useCreateTopNSsmsSetFromFiltersMutation`. Both functions use the passed filter to get a list of gene/ssms ids used to create the set. 

There is also a fix for failing unit test for ColumnOrdering.

Testing note:

One test for this is:

1.  Open Cohort Builder:
2.  Create a cohort with` files.experimental_strategy = WXS`
3.  Open Mutation Frequency
4.  Create Save/Edit Gene Set -> Save as new gene set (with no genes selected) 
5.  The created set should have > 0 genes
also fixes: https://gdc-ctds.atlassian.net/browse/SV-2314

## Checklist

      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
